include/simdjson/padded_string-inl.h

```diff
@@ -185,6 +191,57 @@ inline simdjson_result<padded_string> padded_string::load(std::string_view filen
   return s;
 }
 
+#if defined(_WIN32) && __cplusplus >= 201703L
+  #include <windows.h>
```

Comment:
You are including <windows.h> inside a namespace which is not a good idea. Please move this include at the top of the file.